src/main/java/mods/railcraft/common/items/firestone/ItemFirestoneRefined.java

```diff
@@ -210,4 +213,13 @@ public Entity createEntity(World world, Entity location, ItemStack stack) {
         return entity;
     }
 
+    @Override
+    public void finalizeDefinition() {
+        ItemStack ore = EnumOreMagic.FIRESTONE.getStack();
+        ItemStack rawfirestone = RailcraftItems.FIRESTONE_RAW.getStack();
```

Comment:
Why is this in this class? It doesn't touch the Refined stone at all. Move to Raw?